src/zkevm_specs/evm_circuit/execution/begin_tx.py

```diff
@@ -44,9 +44,10 @@ def begin_tx(instruction: Instruction):
 
     # TODO: Implement EIP 1559 (currently it supports legacy transaction format)
     # Calculate gas fee
+    # sub gas fee from sender balance: gas * gas_fee_cap
```

Comment:
```suggestion
    # Subtract the gas fee from the sender balance: gas * gas_fee_cap
```

You can also remove the TODO above.